src/main/java/org/broadinstitute/hellbender/utils/pileup/PileupElement.java

```diff
@@ -4,11 +4,16 @@
 import htsjdk.samtools.CigarOperator;
 import org.broadinstitute.hellbender.utils.BaseUtils;
 import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.locusiterator.AlignmentStateMachine;
 import org.broadinstitute.hellbender.utils.read.ReadUtils;
 
 import java.util.*;
 
-public final class PileupElement implements Comparable<PileupElement> {
+/**
+ * Represents a position in the read pileup.
```

Comment:
Wouldn't it be more accurate to say `Represents an individual base in a reads pileup`?